codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/AwsProtocolUtils.java

```diff
@@ -49,13 +50,6 @@ static void generateHttpProtocolTests(GenerationContext context) {
                         .name(AddAwsConfigFields.REGION_CONFIG_NAME)
                         .value(writer -> writer.write("$S,", "us-west-2"))
                         .build(),
-                HttpProtocolUnitTestGenerator.ConfigValue.builder()
-                        .name(AddAwsConfigFields.IDEMPOTENCY_TOKEN_PROVIDER)
```

Comment:
Nit: Would want to remove `IDEMPOTENCY_TOKEN_PROVIDER` declaration, if no longer needed.

Reply:
👍